SU-HSDO/suhumsci
# SHS-5692: Implementation: New Spotlight design for Colorful and Traditional

# READY FOR REVIEW

## Summary
Adds new styles for new spotlight design

## Need Review By (Date)
08/09

## Urgency
medium

## Steps to Test
1. In Colorful and Traditional
2. Create a flexible page for Spotlights using the new style "Expanded" with the following considerations:
    - Using different backgrounds (default or gray)
    - Using different heights (short or tall)
    - Aligning the image on the left or right side
    - Spolights in sliders and collections
    - Nested in a colored background (collections with the background option selected)
3. Confirm that they look as expected in both sites: [Traditional Design](https://www.figma.com/design/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?node-id=2308-2584&t=crtPAvWNHYT7bAob-4) / [Colorful Design](https://www.figma.com/design/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?node-id=1731-218&t=crtPAvWNHYT7bAob-4)
4. Please, pay special attention to Warbler color pairing in Traditional
5. Check the "Classic" or default Spotlight style is unchanged

DEMO:
https://www.loom.com/share/a259e04858f94220aa10e26d36fe54d5?sid=d5197c3b-c4fc-4cc6-a0e5-3f53b77afd7a

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208001314957854
